ui/components-react/src/components-react/filter-builder/FilterBuilder.tsx

```diff
@@ -26,6 +26,17 @@ import {
   usePropertyFilterBuilderState,
 } from "./FilterBuilderState";
 import type { PropertyFilter } from "./Types";
+import type { PropertyFilterRuleOperator } from "./Operators";
+
+/**
+ * Props for custom ruleValueRenderer.
+ * @beta
+ */
+export interface CustomPropertyFilterBuilderRuleValueProps
```

Comment:
I think this type should be defined in `FilterBuilderRuleValue` file and maybe rename it to `PropertyFilterBuilderRuleValueRendererProps`

Reply:
fixed